src/clib/solve_rate_cool_g.F

```diff
@@ -329,7 +556,14 @@ subroutine solve_rate_cool_g(icool, d, e, u, v, w, de,
 
 !     Set flag for dust-related options
 
-      anydust = (idust .gt. 0) .or. (idustall .gt. 0)
+      if ((idust .gt. 0) .or. (idustall .gt. 0)) then
+         anydust = MASK_TRUE
+      else
+         anydust = MASK_FALSE
+      endif
+
+!     ignore metal chemistry/cooling below this metallicity
+      min_metallicity = 1.d-9 / z_solar
```

Comment:
In solar units, this is a metallicity of just under 1e-7 Zsun. I suspect I've done this wrong and that the min metallicity is meant to be 1e-9 Zsun. This would be consistent with other places in the code where this is done by hand.